src/main/java/core/basesyntax/service/Writer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface Writer {
+    void createReport(String reportPath);
```

Comment:
The method signature for 'createReport' should include what data it is intended to write to the path provided. For example, it could take a String or a List<String> as an additional parameter that contains the data to be written.